compose_test.go

```diff
@@ -403,7 +431,7 @@ func executeAndGetOutput(command string, args []string) (string, ExecError) {
 	cmd := exec.Command(command, args...)
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return "", ExecError{Error: err}
+		return string(out), ExecError{Error: err}
```

Comment:
Using `cmd.CombinedOutput()` means that stderr is combined in `out`. I added returning it here so that I could detect the `No such volume` error.

Reply:
Neat!